README.md

```diff
@@ -26,6 +33,6 @@ Code licensed under the BSD 3-Clause license. See LICENSE file for terms.
 [issues-image]: https://img.shields.io/github/issues/screwdriver-cd/hyperctl-image.svg
 [issues-url]: https://github.com/screwdriver-cd/hyperctl-image/issues
 [status-image]: https://cd.screwdriver.cd/pipelines/pipelineid/badge
-[status-url]: https://cd.screwdriver.cd/pipelines/pipelineid
+[status-url]: https://cd.screwdriver.cd/pipelines/254
```

Comment:
You might want to go through and remove the badges that aren't relevant, like the NPM badge